addons/mail/static/src/core/common/composer.xml

```diff
@@ -56,7 +56,7 @@
                              the textarea properly without flicker.
                         -->
                         <textarea
-                            class="o-mail-Composer-fake position-absolute"
+                            class="o-mail-Composer-fake px-3 border-0 rounded-3 overflow-hidden position-absolute"
```

Comment:
`form-control` could also have some padding/margin so this can affect sizing too.

We could put all classes in a `t-set` or getter so that we define the shared classnames once and ensure they have the same (with of course some exceptions like `-input/-fake`, `position-absolute` and `overflow-auto/-hidden`)

Reply:
I checked the styling in the browser and seems form-control is fine here, but I can put it back as well.

Indeed, I will use a t-set